src/igvfd/schemas/shared_dependencies.json

```diff
@@ -104,5 +104,34 @@
                 "sources"
             ]
         }
+    },
+    "external_host": {
+        "externally_hosted": {
+            "comment": "Only externally hosted files should have an external host url. External host url is required if the file is externally hosted.",
```

Comment:
I think URL should be all capitalized (may apply in some other places in this PR as well)

Reply:
changed